data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -120,54 +120,6 @@ def _increment_total_eligibility_exceeded(
             axis=1, skipna=True
         )
 
-    def add_columns(self) -> pd.DataFrame:
```

Comment:
Just moving this to the end of the class to follow the convention of putting all private class methods before public class methods. I didn't change any part of the code here.